backend/src/main/java/com/ody/meeting/dto/response/MeetingFindByMemberResponses.java

```diff
@@ -0,0 +1,12 @@
+package com.ody.meeting.dto.response;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+
+public record MeetingFindByMemberResponses(
+        @ArraySchema(schema = @Schema(implementation = MeetingFindByMemberResponse.class))
```

Comment:
[제안]

합의되지 않은 부분이지만, 암묵적으로 파라미터를 내려서 썼다면 개행을 해왔어요.

통일성을 위해 개행을 하는 건 어떨까요? 혹은 이 부분도 합의해보면 좋을 것 같아요.

```suggestion
public record MeetingFindByMemberResponses(

        @ArraySchema(schema = @Schema(implementation = MeetingFindByMemberResponse.class))
```